de.metas.banking/de.metas.banking.swingui/src/main/java/de/metas/banking/bankstatement/match/model/Payment.java

```diff
@@ -72,7 +73,7 @@ private Payment(final I_C_Payment paymentPO, final IPaymentBatch paymentBatch)
 			payAmt = paymentPO.getPayAmt().negate();
 		}
 
-		final I_C_BPartner bpartner = paymentPO.getC_BPartner();
+		final I_C_BPartner bpartner = InterfaceWrapperHelper.load(paymentPO.getC_BPartner_ID(), I_C_BPartner.class);
```

Comment:
note: 
this is just a hint. you can cleanup and polish the whole metasfresh anyways. i would porbably have the same here.
so, just keep it as a hint for *maybe* next time.

if
* you are not going to actually change the bpartner  but just read some of its stuff
* and when the bpartner or any of that stuff was not only just updated in this busine-case which `Payment` is a part of

then do:
```suggestion
		final I_C_BPartner bpartner = InterfaceWrapperHelper.loadOutOfTrx(paymentPO.getC_BPartner_ID(), I_C_BPartner.class);
```
instead. because: caching
or even better: use IBPArtnerDAO

Reply:
Confirmed with @metas-ts there's nothing to do here, since its just compilation fixing.